Src/LexText/LexTextControls/InsertEntryDlg.cs

```diff
@@ -57,6 +57,7 @@ public enum MorphTypeFilterType
 		private ListBox.ObjectCollection m_MGAGlossListBoxItems;
 
 		private Button m_btnOK;
+		private Button m_btnRetry;
```

Comment:
I think `m_btnCreateAndEdit` would be a clearer name (matching the Text rather than the DialogResult).